update on readme.md

ChainYo opened the pull request:
I fixed what it needed to be fixed.

I chose to include in `installation` parts both :
- `bash` command
- link to docs

Tell me if some points need to be removed or updated.

Fixes #1045

jpopelka commented:
/packit build

jpopelka commented on README.md:
```diff
@@ -56,105 +39,25 @@ This list contains workflows covered by packit tool and links to the documentati
 
 Packit is written in python 3 and is supported only on 3.6 and later.
 
-When packit interacts with dist-git, it uses `fedpkg`, we suggest installing it:
```
We either need to remove the below `sudo dnf install -y fedpkg` as well (which I'd prefer) or keep this sentence.

ChainYo replied:
Ok, it's fine to know that it's what you expected. :+1:

jpopelka replied:
Not sure we understand each other :-)
I've been suggesting to remove `sudo dnf install -y fedpkg` as well. (I still see it there in `## Installation`)

jpopelka commented on README.md:
```diff
@@ -1,50 +1,33 @@
+
+## To start using Packit
+
+See our documentation on [packit.dev]
```
```suggestion
See our [documentation](https://packit.dev/docs/guide/)
```

jpopelka commented on README.md:
```diff
@@ -1,50 +1,33 @@
+
+Learn more on [Packit as a service](https://packit.dev/docs/packit-as-a-service/)
+
+Try Packit [Testing Farm](https://packit.dev/docs/testing-farm/)
```
I think we don't need the links to `Packit as a service` and `Testing Farm` here. They are on the website and this readme is just for `packit`.

ChainYo replied:
I added them to make it faster **to get these information for a newcomer**.  
But I can remove them, no problem.

jpopelka reviewed (commented):
Looks OK in general. I'd wait with merging this until https://github.com/packit/packit.dev/issues/197 is fixed.

jpopelka commented:
>     pre-commit: FAILURE in 2m 38s

The `pre-commit` test is failing because you need to rebase your changes.
'rebase' is something you'll need to understand sooner or later anyway if you're going to use git, so I'd suggest investing some time into learning [what it is and how to do it](https://dev.to/matks/what-it-means-to-rebase-a-pull-request-submitted-on-github-5717).

ChainYo commented:
> The `pre-commit` test is failing because you need to rebase your changes.
> 'rebase' is something you'll need to understand sooner or later anyway if you're going to use git, so I'd suggest investing some time into learning [what it is and how to do it](https://dev.to/matks/what-it-means-to-rebase-a-pull-request-submitted-on-github-5717).

I update my fork repository to the new version of the original packit repo. And updated the readme file.  
What is the difference with `git rebase` ? I mean I updated the repo directly on github. I read some infos on `rebase` function but I still don't understand what is the difference.

It seems that I already installed pre-commit packages, but maybe I misunderstood something ! :)

lachmanfrantisek commented:
>  updated my fork repository to the new version of the **original packit repo**. And I updated the readme file by removing the line you quoted on the past comment.
> 
> What is the difference with `git rebase` ? I mean I updated the repo directly on github. I read some infos on `rebase` function but I still don't understand what is the difference.

I am not sure what exactly have you done, but [the article Jirka has linked](https://dev.to/matks/what-it-means-to-rebase-a-pull-request-submitted-on-github-5717) describes the rebase very nicely.

There is really a lot of articles about differences between merge and rebase, but in short:
* Merge will create a new commit that combines changes from your `update_readme` branch and from the new commits from `master` branch.
* Rebase re-applies your commits from your `update_readme` branch on top of the current `master`.


And the effect:
* With merging, there is a lot of merge commits and the git tree is very messy. And if you leave merge of the pull-request to GitHub, we are not testing the same code that will be on master when we merge the pull-request.
* With rebase, the history is clear, we can't have any conflicts with the master branch when we merge pull-request. -- The result is always same as the code in PR.

Just put something like `difference between merge and rebase` to the search engine. You will find a lot of articles and images.

jpopelka commented:
> What is the difference with `git rebase` ? I mean I updated the repo directly on github. I read some infos on `rebase` function but I still don't understand what is the difference.

Yes, there should be no difference.
 
> It seems that I already installed pre-commit packages, but maybe I misunderstood something ! :)

[There are 2 steps](https://github.com/packit/packit/blob/master/.pre-commit-config.yaml#L3), if you did both, you should be fine.
- `pip3 install pre-commit`
- `pre-commit install -t pre-commit -t pre-push`

jpopelka commented:
/packit build

jpopelka reviewed (approved):
Looks OK. Thank you for your contribution! :rocket:

ChainYo commented:
> I am not sure what exactly have you done (probably merge ), but the article Jirka has linked describes the rebase very nicely.

Yes i'm reading it today to understand it better, didn't have time yesterday because of work.

> There is really a lot of articles about differences between merge and rebase, but in short:

Thanks a lot for the quick explanation, I will dig into it more, it seems to be more efficient, so I need to know to do it :)

> There are 2 steps, if you did both, you should be fine.

Yeah I did both.

> Looks OK. Thank you for your contribution! rocket

Thanks for letting me start to help and learn :+1:
